modules/mobile/spec/request/pre_burial_need_request_spec.rb

```diff
@@ -51,14 +51,14 @@
                                               [{ 'title' => 'Validation error',
                                                  'detail' => "The property '#/application/veteran/militaryStatus' of " \
                                                              'type null did not match the following type: string in ' \
-                                                             'schema 47e1937d-acd7-50a1-9cc6-fab567bd62cb',
+                                                             'schema 72d7bc55-042d-5bfd-8001-2b7c815c8e06',
```

Comment:
The schema id for form 40-10007 was updated by [15bda2f](https://github.com/department-of-veterans-affairs/vets-json-schema/commit/15bda2f596f78c0e00b59984cbcef1ff87719d16), due to the update to the common state enum for Canadian provinces.